streamer.c

```diff
@@ -205,14 +205,10 @@ void cLiveStreamer::Action(void)
       // If some additional data is present inside the stream, it is written there (currently RDS inside MPEG2-Audio)
       if (pkt_side_data.data)
       {
-        if (pkt_side_data.streamChange || requestStreamChangeExtData)
+        if (pkt_side_data.streamChange || requestStreamChangeSideData)
           sendStreamChange();
```

Comment:
How is your best way for you about "sendStreamChange();", from my side it must be also removed for now, maybe in the future it becomes needed again on a other way. Was my mistake in your question and answered about this.